consensus/hotstuff_replica.go

```diff
@@ -194,10 +150,25 @@ func (handler *HotstuffReplicaMessageHandler) HandleDecideMessage(m *consensusMo
 }
 
 // anteHandle is the handler called on every replica message before specific handler
+
 func (handler *HotstuffReplicaMessageHandler) anteHandle(m *consensusModule, msg *typesCons.HotstuffMessage) error {
+	log.Println("TODO: Hotstuff replica ante handle not implemented yet")
 	return nil
 }
 
+func (handler *HotstuffReplicaMessageHandler) emitConsensusEvent(m *consensusModule, msg *typesCons.HotstuffMessage) {
```

Comment:
`emitConsensusEvent` is ambiguous. Let's embed the telemetry semantic into the name: `emitTelemetryEvent`. No need to add `Consensus` to the name as we are already under the `consensus` namespace and this method is not exported.

Reply:
Renamed it to `emitTelemetryEvent`